File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

const fileName = 'Agente_IA_Selecao_Curriculos.docx (1).pdf';

if (fs.existsSync(fileName)) {
    let dataBuffer = fs.readFileSync(fileName);
    pdf(dataBuffer).then(function (data) {
        console.log("--- PDF CONTENT START ---");
        console.log(data.text);
        console.log("--- PDF CONTENT END ---");
    }).catch(function (error) {
        console.log("ERROR parsing PDF:", error);
    });
} else {
    console.log("File not found:", fileName);
}

File: read_pdf.py
import sys
from PyPDF2 import PdfReader

try:
    reader = PdfReader('Agente_IA_Selecao_Curriculos.docx (1).pdf')
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"
    print("--- CONTENT START ---")
    print(text[:2000]) # First 2000 chars to review
    print("--- CONTENT END ---")
except Exception as e:
    print(f"Error: {e}")
